data-preprocessing/video-download-upload/credentials.json

```diff
@@ -0,0 +1,12 @@
+{
+  "type": "service_account",
+  "project_id": "nsmq-380115",
+  "private_key_id": "14db2942f1b29fe417317bd2423eb06d03916c99",
+  "private_key": "-----BEGIN PRIVATE KEY-----\nMIIEvAIBADANBgkqhkiG9w0BAQEFAASCBKYwggSiAgEAAoIBAQDQyHIw+wq4fkid\nuqzAwnhtfzWz6A0CcjmKYlJHbR4m5rtz/zeFJK8O6vO+9JiQTc1W+WGQ7JVdV/zm\nWO/4k68Fc5e1lQGdS7n1arOVeqa5G9mDD9OQvEGHGlOZ/AO3ollwZiky/TJIEgeU\nQkEqJNaM+sBkNsRu0VEiO4SLvNfstdJ47OSZEQ8ws7DEMrNIoiHhbOLndGH6H9IX\nzyhAmS8lBeIG76HqIq15mDIvulzK33YgkvE2OV6BhsED4iF+xWy0eW6XQAq0JJi5\nlBAD3r++0FW4fEgh3dx9T/Qt8+MsxPgjl0r35Ib6/2iLzf+AOzj2av23twVyAS2S\nk0Ue0zEdAgMBAAECggEAEcPBy58gthAGDRMG2KPza/BeA7/Ok5MKiFbTWhZ+cUrt\nT50QqU3fbgKEg46mXejw0YyzRjSLL6ppRLmejuUUsv8MkpEU1msiGIVgCxZTkf6S\nTw+NWIxcJasSH/u3M+x8IGSLmYtnuDAoiu8+7N9o42Kr9MWgMy8GRfEQgcoAtA2h\nJxlDB44ccASoOWCciyv/rX0ZsihGSrpzIGg4s/KLzQZkQqnacJQXZNinuN4zGADZ\nktweLuZDqxgULHoqtFH2GsXxmCwq6Wy4rfawoQp6sJcz3uuPD2Vyx1z4MGgAgmq7\no3qGs7vVg0/bgcJ0Kqy9vnbXuRF1tMmSPB5BXtTltQKBgQDpYdHVUb1QhFHF59Vm\nY0N5hchdNDgB289BAjPrXt01jcYcPCSxQhDwpKM1Onlewfhosv9pEHFqwbfh2QIm\nijInnVIaWOO5XiV+hQ/NE6S3FsnlA4EG6qLN6wmhmIJb/T+RZVwlRTiqXSSOfIfW\nPWljUmeItOnP0Ck+JGnM1VFMrwKBgQDlBFNSc2xaHOopgj0bYtuwK629ycshMXxS\n8kl0bE4BajgzMGnHLnbj4yvGyvXrMBCeb2GBTnNZAarh9Slwx5XOUpJyZmdRCIUm\nWs1xVVuTFPxdeyXbyoMEjoWS/b0FPvNRjemi7IGpJY7hwl050A2abMJGJIVAPbB/\nv+rf2i7J8wKBgHnZab3vjGFChJNIhKGNiAglskP/ZJKL2jy90q+7+tDgMaRCPoIR\n1VhfDRtYaU/xBCpd8n4/aIftln0jWT0YADGwsNIywMrifb796A8Rh3lUO5sHZlFf\nJhtO8NtfDgjLQ2o9Oov3BSvlr5V5LU7NB34k4wt8MWsf6riYOGkKowTfAoGALwys\ncYv0/q5iDRUMqMuuhoryrsL+UFtyRzgvjxOREwlXcmzHznvlQPwyyix4SVEemO4Z\nAg+z8t081h8f3GOkiza7j3UqVKODd1XAT0BSB+2Py0S5imYYdJ8zIfevFZzRMDgP\nf8FHAhENQaREstPjd45Qe0iVd5U05YnwHv0cD7ECgYAbPEa9BXZURa74As5mUKHe\n6CkG+jEKttLSU9DOqepVUZm8RhF6w2zS9a6LtV66sbzJLzlz9yMXH7MdSOZ9L+9p\nD6AlLWUu4R90CdXhSggeWgIQ6ZKSLYluERxBvYh7KTCt1QBdAmnc3Pgu34PystJB\nnRNEBvtj+7z5cGGVnhQxkg==\n-----END PRIVATE KEY-----\n",
```

Comment:
let use .env files or other approach to store the private keys. Best practice is to store credentials outside of source control due to security concerns. Please check this link for an example of how to do this https://jonathansoma.com/lede/foundations-2019/classes/apis/keeping-api-keys-secret/